packs/feat-effects/effect-curse-of-ancestral-meddling.json

```diff
@@ -434,4 +434,4 @@
         }
     },
     "type": "effect"
-}
+}
```

Comment:
Please keep the newline at the end of the file.

Reply:
Updated the commit.